src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
```suggestion
    public Color getRandomColor() {
```

if you use Color in class Ball lets return Color from `getRandomColor()` method